### PR TITLE
Refactor ETH test spec

### PR DESCRIPTION
**Pull request by stefan-stefanooov**
**Description**:
Refactor ETH_GetBlockByNumber spec

**Related issue(s)**:
[Refactor eth.spec.ts#1591](https://github.com/hashgraph/hedera-json-rpc-relay/issues/1591)

### CODE REVIEW

**Comment by AlfredoG87** [comment]
Great Refactor, however there are some tests failing 😅

**Comment by stefan-stefanooov** [comment]
> Great Refactor, however there are some tests failing 😅

Yes, sir. Working on it.

**Comment by stefan-stefanooov** [comment]
@AlfredoG87 I think it's ready :)

File: packages/relay/tests/lib/eth/eth_estimateGas.spec.ts [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+import { generateEthTestEnv } from './eth-helpers';
+
+dotenv.config({ path: path.resolve(__dirname, '../test.env') });
+use(chaiAsPromised);
+
+let sdkClientStub;
+let getSdkClientStub, ethImplOverridden;
+let currentMaxBlockRange: number;
+
+describe('@ethEstimateGas Estimate Gass spec', async function () {
```
Comment (AlfredoG87) [inline packages/relay/tests/lib/eth/eth_estimateGas.spec.ts:42]:
```suggestion
describe('@ethEstimateGas Estimate Gas spec', async function () {
```

Typo?

**Review by AlfredoG87 — DISMISSED** [review]
Only found one nit, a typo.
LGTM

**Review by konstantinabl — APPROVED** [review]

**Pull request merged**